library/Icingadb/Hook/ActionsHook/ObjectActionsHook.php

```diff
@@ -61,6 +61,10 @@ final public static function loadActions(Model $object): HtmlElement
                         continue;
                     }
 
+                    if (! $link->hasAttribute('data-base-target') && ! $link->hasAttribute('target')) {
+                        $link->setAttribute('data-base-target', '_next');
```

Comment:
We're dealing with `ipl\Web\Widget\Link` here, so please use `getBaseTarget()` and `setBaseTarget()`